donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends "pages/base_page.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Ways to Give" %}{% endblock %}
+
+{% block template_name %}app--ways-to-give{% endblock %}
+
+{% block header %}
+<div class="app__header">
+    <h1>{% trans "Ways to Give" %}</h1>
+    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "donate now" %}</a></div>
+</div>
+{% endblock %}
+
+{% block navigation %}{% endblock %}
+
+{% block content %}
+<div class="container">
+    <h2>{% trans "Credit / Debit and PayPal" %}</h2>
+    <p>
+        {% blocktrans trimmed %}
+        <a href="{{ways_to_give_link}}">Go to this link</a> to make a secure online donation right now.
+        {% endblocktrans %}
+    </p>
+
+    <h2>{% trans "Bank Transfer / SEPA" %}</h2>
+    <p><strong>{% trans "New!" %}</strong> {% trans "We are now able to accept donations in Euros through Single European Payment Area (SEPA) transfer, and British Pounds through BACS transfer." %}</p>
```

Comment:
This is not new anymore, let’s use the copy we have here instead https://give.mozilla.org/en-US/ways-to-give/